cutplace/interface.py

```diff
@@ -514,8 +515,6 @@ def as_sql_create_table(self, dialect='ansi'):
         create_table = "create table " + table_name[0] + " (\n"
```

Comment:
As this is the variable holding the function result rename it to `result` to make that clear.